.github/workflows/package-Instrumentation.StackExchangeRedis.yml

```diff
@@ -1,4 +1,4 @@
-name: Pack OpenTelemetry.Instrumentation.StackExchangeRedis
+Profilername: Pack OpenTelemetry.Instrumentation.StackExchangeRedis
```

Comment:
@utpilla What's up with this "Profiler" bit? Copy/paste fail?

Reply:
Great catch! And yes looks like a terrible copy/paste fail 😂